libs/components/theme/src/lib/styles/_theme.scss

```
@@ -141,6 +141,27 @@ symbol[id^='sky-i-']:not(.sky-i-branded) * {
   }
 }
 
+svg.sky-illustration-svg {
```

Is the `svg` element qualifier needed? Do we use the `sky-illustration-svg` class on non-SVG elements where we don't want to apply these rules?

```suggestion
.sky-illustration-svg {
```